prepare_karthikeyan97.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
The file (currently) needs execute permissions, otherwise the `test.sh` script will not use it (-> @gunnarmorling, this could be enhanced in test.sh as well).

Reply:
BTW: Executing test.sh reveals an error
```
Validating calculate_average_karthikeyan97.sh -- src/test/resources/samples/measurements-1.txt
1c1
< 
---
> Kunming;19.8;19.8;19.8
```